bouncer/util.py

```diff
@@ -61,6 +76,9 @@ def parse_document(document):
         except KeyError:
             pass
 
+    if quote == "":
+        quote = "Annotation for {}".format(document_uri)
```

Comment:
Do we need a to show a text if there is no exact quote selector? This wouldn't look so great for a complicated url: "Annotation for https://www.cambridge.org/core/books/history-manifesto/big-questions-big-data/F60D7E21EFBD018F5410FB315FBA4590/core-reader"

Depending on how Facebook and Twitter render the cards, it might look nicer to just not add the `description` meta tag in this case?